<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPortalWizard</class>
 <widget class="QWizard" name="AddPortalWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPage0">
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="editServer">
      <property name="text">
       <string>http://127.0.0.1:8080/</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblServer">
      <property name="text">
       <string>Server</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editTitle">
      <property name="text">
       <string>test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Select Authentication</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cbAuthSelector"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="lblUsername">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editUsername"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editPassword"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblPassword">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="cbSavePw">
      <property name="text">
       <string>Save Password?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage3">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Success</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
